src/resolvelib/resolvers.py

```diff
@@ -369,7 +369,7 @@ def resolve(self, requirements, max_rounds):
                 causes = [i for c in failure_causes for i in c.information]
                 # Backtrack if pinning fails. The backtrack process puts us in
                 # an unpinned state, so we can work on it in the next round.
-                self._r.start_backtracking(causes)
+                self._r.resolving_conflicts(causes)
```

Comment:
```suggestion
                self._r.resolving_conflicts(causes=causes)
```

All provider and reporter calls should use named arguments for forward compatibility.